source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Autodefine External Network
+category: feature
+authors: Ales Musil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+Engine automatically creates an external network based on physical VM network
+with physical connection to it. 
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Autodefine helps reducing administrative tasks needed with OVS switch type in 
+the cluster.
+
+## Prerequisites
+
+Cluster has to be configured with switch type OVS and valid external provider.
+The user needs to make sure that connection to external provider is valid.
+
+### Limitations
+
+* With OVS switch type in cluster, only external networks are allowed to 
+  be attached to the VM interface. 
+
+* Physical network linked to any external network can't be removed. 
+
+* Existing Clusters switch cannot be changed to OVS. Currently there is no way 
+  in the Engine to follow this change across all networks and VM interfaces.
+
+## Detailed description
+
+### Creating of new physical network
+
+When creating new network choose its name. Check the `VM network`. If vlan 
+tagging is needed, check the `Enable VLAN tagging` and put the tag into the 
+field. Please note that auto-define can be triggered only on top of network that
+is not marked as external. 
+
+![Creation of physical network](../../../../images/wiki/auto-define_create.png  "New physical network")
+
+After confirmation if all conditions were met, the external network connected to 
+the physical one, will be created simultaneously. The auto-defined external 
+network is named `external_name_of_physical_network` (The physical network name
+might be shortened so that auto-defined network name is not bigger than 256 
+characters). 
+
+![Auto-define network created](../../../../images/wiki/auto-define_created.png  "Auto-define external network")
+
+In case there are clusters with different default external providers the Engine 
```

Comment:
In case there are clusters with different default external providers **in the data center** the Engine will create multiple external network**s** for every provider and add **a** number **to** this auto-defined network name.

Reply:
Done